### ISSUE TITLE
UI tweaks in Wikibase selection dialog

### ISSUE THREAD

**Issue by wetneb** [OpenRefine/OpenRefine#3824]
I would like to propose some minor tweaks to the dialog to select the Wikibase instance:
* The currently selected Wikibase instance is not displayed prominently enough. I would expect the item in the list to be somehow highlighted (different background color, border…)
* Since the logos take a bit of vertical space I think we could have space to also display the URL of the Wikibase instance underneath the name. I expect it could be useful to distinguish between various instances with similar names/logos.
* When I click on another Wikibase instance to select it, I find it confusing that the dialog does not go away by itself. What else could I want to do in this dialog after that? Its only purpose is to select the Wikibase instance, so once that is done I should be able to just go back to the project and carry on.

Pinging @afkbrb for his thoughts on this :)

**Comment by afkbrb** [comment]
The first and third points are very reasonable, but I'm kind of confused by the second point.

> Since the logos take a bit of vertical space I think we could have space to also display the URL of the Wikibase instance underneath the name. I expect it could be useful to distinguish between various instances with similar names/logos.

Do you mean we should include both the logo and the URL for each item? I think adding the URL of the Wikibase instance is enough to distinguish between various instances with similar names/logos, and adding the logo is not necessary and it will require the URL of the logo to be included in the manifest.

**Comment by wetneb** [comment]
Ah, maybe you didn't see that we'll already be displaying the logos once https://github.com/OpenRefine/OpenRefine/pull/3702 is merged (sorry, I anticipated that a bit!).

**Comment by afkbrb** [comment]
The second point is definitely reasonable then :)

**Comment by elebitzero** [comment]
> * When I click on another Wikibase instance to select it, I find it confusing that the dialog does not go away by itself. What else could I want to do in this dialog after that? Its only purpose is to select the Wikibase instance, so once that is done I should be able to just go back to the project and carry on.

@wetneb,  I've implemented items # 1 and # 2, but not this last point, because this dialog is also used to add and delete Wikibase instances, and we could add other features like clicking to go to the URL.   This needs further design work, like separating the selection into a simple selection dropdown, and then having the configuration in a dialog.

Question: Should the displayed URL be mediawiki.root or mediawiki.main_page ?

**Comment by wetneb** [comment]
That is a good point.

If I remember correctly, I think @lozanaross thought about the issue of selection of the Wikibase instance. At the moment, the way we indicate the currently selected Wikibase instance in the schema is not really visible (because it is buried in an explanation sentence). One option that was considered was to have a drop-down menu inside the schema editor to select the Wikibase instance, which would let us solve multiple problems at once:
* the currently selected Wikibase instance would be more visible (since it would appear "framed", as the content of a drop-down input)
* it would also be faster to change Wikibase instance (no need to open a dialog for that)
* the dialog where we list Wikibase instances could be used just to manage the list of available instances (add a new one, delete an existing one)

What do you think about it?

**Comment by lozanaross** [comment]
Hello all - sorry for slowness in response here. Indeed, I have been working on some improvements in this area in the SDC extension wireframes, so it makes sense to combine efforts. 

Here is the updated design for the "Select Wikibase" dialog in the SDC wireframes:
![SDCWireframes_WikibaseInstanceSelection](https://user-images.githubusercontent.com/25489750/163805780-8e0cc1c6-9dca-4b3a-a7b1-cf218a897ae8.png)

What is changed here is updated and simplified copy at the top (based on user feedback), addition of radio button (I've added URLs, too, though not sure if those are super necessary), blue highlight color & change in the order of button options in the footer part of the dialog. 

And here is the simplified way to "switch" across target instances in the Schema tab:
![SDCWireframes_SchemaBuilder](https://user-images.githubusercontent.com/25489750/163805835-ea8cd982-5c22-407f-8d08-e0981540a3c1.png)

What is changed here is the ability to switch WB instances from a drop down menu & for some instances, eg Commons, if there are templates available then that second dropdown menu will also become available. Previous long copy is removed, because detailed instructions can be provided in user onboarding materials, no need to be always there.

**Comment by elebitzero** [comment]
@wetneb, I like the idea of having a quick select menu for the Wikibase, but I think that is a bigger change and deserves its own separate issue.  

My PR is just focused on fixing the UX issues (points #1 and #2) keeping within the current design.
